README.md

```diff
@@ -46,24 +46,23 @@ Running locally outside an OKD cluster:
  $ make run
 ```
 ### `make` targets
-Various `make` targets are included to simplify building and deploying the operator
-from the repository root directory, all of which are not listed here.  Hacking and
-deploying the operator assumes:
-* a running OKD cluster
-* `oc` binary in your `$PATH`
-* Logged into a cluster with an admin user who has the cluster role of `cluster-admin`
-* Access to a clone of the `openshift/elasticsearch-operator` repository
-* `make` targets are executed from the `openshift/cluster-logging-operator` root directory
-* various other commands such as `imagebuilder` and `operator-sdk`. The
-   [sdk_setup.sh](https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/hack/sdk_setup.sh) will setup your environment.
+The main `make` targets are listed below, see the Makefile for more.
+
+Hacking and deploying the operator assumes:
+* Logged into an OKD cluster with cluster role `cluster-admin`
+* Clone of `openshift/elasticsearch-operator` repository in `$GOPATH`
+* `make` run from this directory `openshift/cluster-logging-operator`
+* Commands such as `oc`, `imagebuilder`, `operator-sdk` and `golangci-lint`.
+  NOTE: If a command is not found in `$PATH`, or in `./bin`, `make`
+  will automatically download or build the command in `./bin`.
+
+See also: [sdk_setup.sh](https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/hack/sdk_setup.sh) to set up your environment.
 
 The deployment can be optionally modified using any of the following:
 
 | Env Var | Default | Description|
 |---------|---------|------------|
-|`IMAGE_BUILDER`|`imagebuilder`| The command to build the container image|
 |`EXCLUSIONS`|none|The list of manifest files that should will be ignored|
-|`OC`|`oc` in `PATH`| The openshift binary to use to deploy resources|
 |`REMOTE_REGISTRY`|false|`true` if you are running the cluster on a different machine than the one you are developing on|
```

Comment:
While you are updating, can you remove the ref here and below to "REMOTE_REGISTRY" as they are not needed atm